compiler/fir/fir2ir/src/org/jetbrains/kotlin/fir/backend/Fir2IrTypeConverter.kt

```diff
@@ -54,6 +55,7 @@ class Fir2IrTypeConverter(
 
     fun FirTypeRef.toIrType(typeContext: ConversionTypeContext = ConversionTypeContext.DEFAULT): IrType {
         return when (this) {
+            is FirJavaTypeRef -> coneType?.toIrType(typeContext) ?: createErrorType()
```

Comment:
Yet it's Ok but I should investigate this place. As I understand we should not have `FirJavaTypeRef` at this point, and if we do, it points to some FIR resolve error.